core/iwasm/libraries/ckpt-restore/ckpt_restore.h

```diff
@@ -0,0 +1,143 @@
+/*
+ * The WebAssembly Live Migration Project
+ *
+ *  By: Aibo Hu
+ *      Yiwei Yang
+ *      Brian Zhao
+ *      Andrew Quinn
+ *
+ *  Copyright 2024 Regents of the Univeristy of California
+ *  UC Santa Cruz Sluglab.
```

Comment:
Could you add the license info in the source files of this folder? And do you agree them to use the license of this project, or add a line `SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception` like other files?